tests/benchdnn/reorder/reorder.cpp

```diff
@@ -354,6 +354,26 @@ void skip_unimplemented_prb(const prb_t *prb, res_t *res) {
             res->reason = skip_reason::case_not_supported;
             return;
         }
+
+        // Skip blocked format tags and 4 bit formats for Nvidia/AMD/Generic SYCL backends
+        const auto &engine = get_test_engine();
+
+        if (is_generic_gpu(engine) || is_nvidia_gpu(engine)
+                || is_amd_gpu(engine)) {
```

Comment:
NVidia and AMD backends are skipped by default.
```suggestion
        if (is_generic_gpu()) {
```